ui-core/src/components/inputs/Slider.tsx

```diff
@@ -19,12 +19,17 @@ const Slider = ({
   onChange,
   onChangeCommitted,
   disabled,
+  className,
   ...rest
 }: SliderProps) => {
   const [value, setValue] = useState<number>(
     initialValue !== undefined ? Number(initialValue) : Number(min)
   );
 
+  useEffect(() => {
+    setValue(Number(initialValue));
+  }, [initialValue]);
```

Comment:
Hm, it's a bit weird to have this internal state. Would be nicer to make this a controlled component like other form inputs, and have a single source of truth when it comes to the value.

But that sounds like a larger change.

Reply:
This internal state allows to do the `onChangeCommitted` that only update after the user releases the mouse.